cirq/ops/common_gates_test.py

```diff
@@ -74,9 +74,8 @@ def test_cz_matrix():
                                  [0, 0, 1, 0],
                                  [0, 0, 0, -1j]]))
 
-    cirq.testing.assert_apply_unitary_to_tensor_is_consistent_with_unitary(
-        val=cirq.CZ,
-        exponents=[1, -0.5, 0.5, 0.25, -0.25, 0.1, cirq.Symbol('s')])
+    cirq.testing.assert_has_consistent_apply_unitary_for_various_exponents(
+        cirq.CZ)
```

Comment:
I think it's confusing that these tests don't use `assert_eigen_gate_has_consistent_apply_unitary`, since they are EigenGates after all. I think we should get rid of all the overriding `__init__` methods on the Pow gates and just have the one test method.